packages/@datapull/kinesis-firehose-destination/util/batch-splitter.js

```diff
@@ -0,0 +1,43 @@
+module.exports = function (records, callback) {
```

Comment:
As far as I can tell this is a direct copy of `kinesis-stream-destination`'s `batch-splitter`. Why not have them put centrally and re-use them?

Reply:
This request introduces an ability to send messages to Kinesis Firehose, which has a similar, but still not the same APIs, that have their own considerations. So I would like to introduce a separate module to publish messages to Firehose, as opposed to having one which can produce for Kinesis Streams and Firehose based on the pipeline configuration. In fact, I found maintaining one unified producer to be harder to maintain, thus there is an idea of splitting these apart. You can already see that batch-splitter is not the direct copy and has some differences between the two, and in the future, it would be easier to introduce changes to both of the destination modules if they are completely separated from each other.